doc/fr/configuration_guide/advanced_configuration/dependancy.rst

```diff
@@ -34,14 +34,7 @@ Il est possible de définir deux paramètres :
 .. note:: 
     Tous les parents d’un hôte doivent être dans un état non-OK pour que l’hôte lui-même soit considéré comme injoignable. A partir du moment où au moins un chemin d’accès (liaison de dépendance physique) est toujours joignable alors l’ordonnanceur continuera de surveiller cet hôte.
 
-Dans le cas où des relations de parentés ont été définies entre hôtes supervisés par des ordonnanceurs différents,  il est possible :
-
-* D'empêcher l'établissement d'une relation de parenté, lors de la modification du formulaire d’hôte, entre deux hôtes supervisés par deux collecteurs différents.
-* D'autoriser l'établissement de cette relation de parenté. Dans ce cas la dépendance ne sera pas gérée par les moteurs de supervision mais par Centreon Broker qui prendra en compte cette relation au sein de son moteur de corrélation.
-
-Pour empêcher l'établissement de cette relation de parenté, il est nécessaire de cocher la case **Activer le mode strict de gestion des relations de parentés** au sein du menu **Administration > Options**.
-
-A l'inverse si cette case n'est pas cochée alors les liens de parenté entre hôtes appartenant à deux collecteurs différents peuvent être établis.
+Il n'est pas possible d'établir des liens de parentés entre hôte supervisés par deux collecteurs différents.
```

Comment:
```suggestion
Il n'est pas possible d'établir des liens de parentés entre des hôtes supervisés par deux collecteurs différents.
```

Reply:
Could replace "par deux collecteurs" by "par des collecteurs"
sorry for the typos :) !!